### PR TITLE
docs: updated readme example code and added link to spec

### LINKED ISSUE #14: Readme.md code example is out of date
~~Currently the code example in the README.md does not include the first argument being the defined sassNull, which is required in the current module API.~~ Disregard that -- I was reading an out of date version somehow :-(. The below still holds!

It also includes a ``;`` inside the options object, which means it will not run, and prompts the console-logging of ``sass.render``, which will be undefined (as that is an async function).

A little bit of cleanup will help make this more consumer friendly :)

### PR DESCRIPTION
**Pull request by edragic**
Resolves #14 
- removed errant `;`
- ensured example sass command would log css
- updated with link to Roux pantry spec

### CODE REVIEW

**Comment by edragic** [comment]
When working on updating an old project, something that cost me time was not realizing that it was important to have the

``
  "roux": {
    "pantryRoot": "pantry"
  }
``
which is why I also added the note here.

**Comment by edragic** [comment]
hmm, it looks like this PR is generating an eslint error on build about tabs/spaces in the index.js file, but this changeset only hits the README.md. Not sure where whitespace errors would've entered.

**Comment by elyseholladay** [comment]
@lzilioli @edragic what do we need to do to land this? are y'all still seeing whitespace issues?

**Comment by edragic** [comment]
ah we decided I should just fix the whitespace check that seemed to have never worked. but then I forgot because I'm dumb. 

Just pushed.

File: README.md [hunk 1]
```diff
@@ -36,17 +36,20 @@ var importer = rouxSassImporter(sass.NULL, {
...
+sass.render({
+  file: 'path/to/your/file.scss',
+  importer: importer
+},
+function(err, result){
+  console.log(result && result.css.toString());
+});
 ```
 
+If passing only a path, and you have nested the pantry entry, please ensure that you have correctly notes that in your package.json, per the [roux README][pantry-root-readme].
```
Comment (elyseholladay) [inline README.md]: "correctly **noted**" typo

File: README.md [hunk 2]
```diff
@@ -36,17 +36,20 @@ var importer = rouxSassImporter(sass.NULL, {
   pantrySearchPaths: [
     path.resolve('node_modules'),
     path.resolve('some/other/path')
-  ];
+  ]
```
Comment (elyseholladay) [inline README.md]: is there a code edit that needs to be done to match this, or was this just a mistake in the docs?

Comment (edragic) [inline README.md]: good call out -- this was just a mistake in the example code about how to use this module. 

I realized it when I went to check something and got a lint error when I grabbed this as a starting point, as it stands the example js won't run

File: README.md (hunk 1 above)
Comment (edragic) [inline README.md]: oh whoops...😳. will fix!

**Comment by edragic** [comment]
@elyseholladay fixes are pushed now! thanks for the review :)

**Comment by elyseholladay** [comment]
LG to me, @lzilioli?

**Comment by lzilioli** [comment]
LGTM

**Pull request merged**